src/components/HypothesisBuilder/components/CommentsSection/index.js

```diff
@@ -90,16 +73,15 @@ export function CommentsSection({
                       />
                     </span>
                   </div>
-                  <div className="min-w-0 flex-1">
+                  <div className="min-w-0 flex-1 text-inter">
```

Comment:
please remove `text-iter` typology changes should be at the tailwind.config.css level not at the component level